src/components/AccountManager/AccountManager.js

```diff
@@ -0,0 +1,253 @@
+                </InputGroup>
+                <br />
+                <InputGroup className={"inputGroupLogin"}>
+                    <InputGroupAddon className={"pre-pend"} addonType="prepend">Email</InputGroupAddon>
+                    <Input className={"inputField"} type="email" placeholder="Email..." onChange={(e) => setChangeEmail(e.target.value)}/>
+                </InputGroup>
+                <br />
+                <Button color="danger" onClick={changeInfo}>{confirmationText}</Button>
+            </Collapse>
+
+            <Collapse isOpen={isResetPasswordOpen}>
+                <label>{resetPasswordOne} :::WIP::: {resetPasswordTwo}</label>
+                <br />
+                <label>{passwordsMatch}</label>
+                <InputGroup className={"inputGroupLogin"}>
+                    <InputGroupAddon className={"pre-pend"} addonType="prepend">New Password</InputGroupAddon>
+                    <Input className={"inputField"} type="password" placeholder="..." onChange={(e) => setResetPasswordOne(e.target.value)}/>
+                </InputGroup>
+                <br />
+                <InputGroup className={"inputGroupLogin"}>
+                    <InputGroupAddon className={"pre-pend"} addonType="prepend">New Password</InputGroupAddon>
+                    <Input className={"inputField"} type="password" placeholder="..." onChange={(e) => setResetPasswordTwo(e.target.value)}/>
+                </InputGroup>
+                <br />
+                <Button color="danger">{confirmationText}</Button>
+            </Collapse>
+
+            <hr />
+
+            <Button color="info" onClick={toggleFaceSetup} style={{ marginBottom: '1rem' }}>Face Login Setup</Button>
+            <Collapse isOpen={isFaceSetupOpen}>
+                {
+                    !isGuest ?
+                        <FaceLoginSetup/>
+                        :
+                        <label>DISABLED AS GUEST</label>
+                }
+            </Collapse>
+
+            <hr />
+
+            <Button color="warning" onClick={toggleGoogle} style={{ marginBottom: '1rem' }}>Google Calendar Config</Button>
+            <Collapse isOpen={isGoogleOpen}>
+                {
+                    !isGuest ?
+                        <GoogleCalendarConfig/>
+                        :
+                        <label>DISABLED AS GUEST</label>
+                }
+            </Collapse>
+
+            <hr />
+
+            {
+                !isGuest ?
+                    <Button color="danger" onClick={deleteAccount}>Delete Account</Button>
+                    :
+                    <></>
+            }
+
+            <Alert color="danger" isOpen={visibleDeleteConfirmation} toggle={onDismissDeleteConfirmation}>
+                <label>Are you sure? </label>
+                &nbsp;
+                <Button color="danger" onClick={deleteAccountConfirmation}>Confirm</Button>
```

Comment:
It's safer to use anonymous functions for event handlers like this: `onClick={ () => deleteAccountConfirmation() }`.

When passing the function directly sometimes it be called automatically when the component renders

Reply:
Changed.